README.md

```diff
@@ -243,9 +242,9 @@ Run MEV-Boost pointed at a Sepolia relay:
 
 These are the CLI arguments for the main branch. For arguments available in a specific release, check the [release page](https://github.com/flashbots/mev-boost/releases).
 
-```
-$ ./mev-boost -help
-Usage of ./mev-boost:
+```bash
+$ mev-boost -help
```

Comment:
I think this should have the ./ prefix for consistency.

Reply:
https://github.com/flashbots/mev-boost/pull/420